### PR TITLE
Test cases

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request adds tests which cover releases test cases.
In particular, there are two new tests, one which checks the installation directory, and others which checks the packaging and compile an example.

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/19/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/18/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/20/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/19/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/21/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/20/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/22/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/22/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/23/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/23/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -21,11 +21,16 @@ cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
 # Build options
 ###############################################################################
 option(UCDR_SUPERBUILD "Enable superbuild compilation." ON)
-option(UCDR_BUILD_TESTS "Build tests" OFF)
+option(UCDR_BUILD_TESTS "Build tests." OFF)
 option(UCDR_BUILD_EXAMPLES "Build examples" OFF)
```
Comment (BorjaOuterelo) [inline CMakeLists.txt]:
```suggestion
option(UCDR_BUILD_EXAMPLES "Build examples." OFF)
```

File: ci/linux/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    CMAKE_CACHE_ARGS
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DCMAKE_C_FLAGS:STRING=${_c_flags}
+        -DCMAKE_CXX_FLAGS:STRING=${_cxx_flags}
+        -DCMAKE_EXE_LINKER_FLAGS:STRING=${_exe_linker_flags}
+        -DCMAKE_SHARED_LINKER_FLAGS:STRING=${_shared_linker_flags}
+        -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
+        -DUCDR_BUILD_CI_TESTS:BOOL=ON
+        -DGTEST_INDIVIDUAL:BOOL=ON
+    )
```
Comment (BorjaOuterelo) [inline ci/linux/CMakeLists.txt]: Shall we adopt the same changes regarding Ninja and multiple compilers done in Client?

Comment (julianbermudez) [reply ci/linux/CMakeLists.txt]: Why not?

**Pull request merged**